src/Library/demos/Drop Down/main.blp

```diff
@@ -0,0 +1,73 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: _("Drop Down");
+
+  Box {
+    orientation: vertical;
+    halign: center;
+    valign: center;
+
+    Grid  {
```

Comment:
Can you avoid using Grid unless it's necessary? 

It makes `UI` a bit more complicated than it needs to be.

Reply:
It may also be possible to use a set of horizontal boxes with some appropriate properties, if it makes the demo easier to read. That's definitely something to keep in mind.

I usually prefer a `GtkGrid` myself, but I'm more concerned about the "jumping around" part.

Reply:
oh, well I already removed it. I just gave the gave the dropdowns a width-request so they wouldn't jump around